src/stories/SecondaryContentBanner.stories.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { checkA11y } from '@storybook/addon-a11y';
+import { MemoryRouter } from 'react-router-dom';
+import { FormattedMessage, IntlProvider } from 'react-intl';
+
+import SecondaryContentBanner from 'js/page_sections/SecondaryContentBanner';
+import ExternalLink from 'js/modules/ExternalLink';
+
+storiesOf('SecondaryContentBanner', module)
+  .addDecorator(checkA11y)
+  .addDecorator(story => (
+    <MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>
+  ))
+  .add('SecondaryContentBanner', () => (
+    <IntlProvider locale="en">
+      <SecondaryContentBanner>
+        <p>
+          <FormattedMessage
```

Comment:
Maybe we can put dummy text in, instead of FormattedMessage react-intl component. It's not required to test the component.